src/helpers/text_helper.ts

```diff
@@ -21,12 +24,22 @@ export function computeTextLinesHeight(textLineHeight: number, numberOfLines: nu
 export function getDefaultCellHeight(
   ctx: CanvasRenderingContext2D,
   cell: Cell | undefined,
+  locale: Locale,
   colSize: number
 ) {
   if (!cell || (!cell.isFormula && !cell.content)) {
     return DEFAULT_CELL_HEIGHT;
   }
-  const content = cell.isFormula ? "" : cell.content;
+  let content = "";
+
+  try {
+    if (!cell.isFormula) {
+      const localeFormat = { format: cell.format, locale: DEFAULT_LOCALE };
+      content = formatValue(parseLiteral(cell.content, locale), localeFormat);
+    }
+  } catch {
+    content = CellErrorType.GenericError;
```

Comment:
can this fail for other reasons than an invalid format? 
and is it possible in real life conditions to have an invalid format?